server/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#
+# Copyright 2017 The Chromium OS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Runs a server that receives log uploads from the WALT app
+Usage example:
+    $ python main.py
+"""
+
+import time
+import os
+from bottle import route, template, run, request, static_file
+
+@route('/')
+def index():
+    if not os.path.isdir("logs/"):
```

Comment:
Keep consistent with single/double quote usage.
PEP8 has no recommendation on which to use as long as it's consistent, but in pywalt it's single quotes.
https://www.python.org/dev/peps/pep-0008/#string-quotes